ams/server/src/main/java/com/netease/arctic/server/DefaultOptimizingService.java

```diff
@@ -95,6 +96,17 @@ public void loadOptimizingQueues(List<TableRuntimeMeta> tableRuntimeMetaList) {
     groupToTableRuntimes.keySet().forEach(groupName -> LOG.warn("Unloaded task runtime in group " + groupName));
   }
 
+  public void loadOptimizers() {
```

Comment:
`loadOptimizers()` doesn't put optimizers to the `authOptimizers` of `OptimizingQueue`.